src/app/researcher/study/request/[memberIdentifier]/study-proposal-form-steps.tsx

```diff
@@ -25,6 +26,11 @@ export const StudyProposalFormSteps: React.FC<{ memberId: string }> = ({ memberI
         onSuccess() {
             router.push(`/researcher/dashboard`)
         },
+        onError(error) {
+            // TODO server action max filesize error doesn't propagate through here...
+            console.error(error)
+            notifications.show({ message: String(error), color: 'red' })
+        },
```

Comment:
this is great, but I don't think we should show raw server errors to the user, because it may contain sensitive info.  

`lib/errors.ts` has an errorToString that filters them, and `lib/components/errors.tsx` has a `reportError` method that uses it and also logs to sentry.   

tldr; use `reportError` vs calling notifications.show directly 😁